charts/vela-core/templates/defwithtemplate/addon-operation.yaml

```
@@ -22,7 +22,7 @@ spec:
         		apiVersion: "batch/v1"
         		kind:       "Job"
         		metadata: {
-        			name:      parameter.addonName + "-enable-job"
+        			name:      context.stepName + context.name + context.namespace + "-enable-job"
```

1. namespace is not necessary, it will contain in the job namespace.

```suggestion
        			name:     context.name + "-" + context.stepName"-enable-addon"
```